texthero/representation.py

```diff
@@ -180,17 +88,18 @@ def count(
     --------
     >>> import texthero as hero
     >>> import pandas as pd
-    >>> s = pd.Series(["Sentence one", "Sentence two"]).pipe(hero.tokenize)
-    >>> hero.count(s)
-    0  Sentence    1
-       one         1
-    1  Sentence    1
-       two         1
-    dtype: Sparse[int64, 0]
-
+    >>> s = pd.Series(["Sentence one", "Sentence two"]).pipe(
+    ...                                    hero.tokenize
+    ...                                                )
+    >>> hero.count(s) # doctest: +SKIP        
```

Comment:
remove # doctest: +SKIP

Reply:
I'll try that but I'm pretty sure this will sadly fail on MacOS in travis as it prints the DF differently

Reply:
OK. As a general comment: we need to find a general stable solution; this problem is taking us too much time. The initial idea of having doctests was to just make sure that the output of the Example section corresponded to the output of the Texthero functions; that's it. Say we change a hero function, doctests help us make sure we update also the docstring Example section accordingly.

I believe that it make sense somehow to make sure that the output of the Examples corresponds to the output of the functions, yet, we don't wanto to get crazy if the outputs from mac and windows change just a bit or if we forget a whitespace somewhere.

What I propose is to stop testing the doctest as we are doing now and write our own wrapper to test that. By looking at the unittest and using for instance [inspect](https://docs.python.org/3/library/inspect.html) I think this will be feasible. We might want also to write a "tech" article on the blog page or something like that as this might be useful for others too. 

What are your opinions and ideas?

Reply:
I agree that the doctests are annoying. I'll think about this some more - it makes sense to not use the default module as it gets really annoying now. I'll look into alternatives like e.g. a decorator